frontend/src/models/Step.ts

```diff
@@ -1,12 +1,17 @@
 import Status from './Status';
 
+export interface StepRow {
+  cells: string[];
```

Comment:
I'm a bit confused what this is - cells and line don't really sound like 'step'-related things, in fact 'rows' doesn't sound very step-related either... What actually are these and are there some better names we could give them?

Reply:
I have no idea tbh, this is Feier magic. I will look into it more today but I dare say we either need to change the step model or it is being processed into that in a file I haven’t converted yet.

Reply:
As discussed this is actually a thing coming from the backend/junit reporting